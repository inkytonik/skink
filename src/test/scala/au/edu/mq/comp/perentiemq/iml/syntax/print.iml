def printing(a): (b) {
  print (a);
  print (a + b);
}
