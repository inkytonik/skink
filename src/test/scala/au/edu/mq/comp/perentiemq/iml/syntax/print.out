Program (
    List (
        FunDef (
            "printing",
            List (Arg ("a")),
            List (Arg ("b")),
            Block (
                List (
                    Print (ELoad (Var ("a"))),
                    Print (Add (ELoad (Var ("a")), ELoad (Var ("b")))))))))
def printing(a): (b) {
    print (a);
    print (a + b);
}

